### ISSUE TITLE
Add usage instructions to README and help.

### ISSUE THREAD

**Issue by MicahZoltu** [Arachnid/evmdis#15]
I build/installed evmdis and ran `evmdis -h`.  It shows me flags I can pass but doesn't actually tell me how to use it.  I took a guess and did:
`echo <some evm bytecode> | evmdis` but that resulted in:
```
# Stack: []
0x0     STOP()
```
Which doesn't seem correct.  Readme should have usage instructions, and the CLI help should as well.

**Comment by Arachnid** [comment]
Yes, usability needs big improvements. For reference, try `echo -n`; one of the many current shortcomings is very picky hex parsing.

**Comment by bohendo** [comment]
If anyone is analyzing an output file that might contain a newline, you can also do the following:

```
cat file.bytecode | tr -d '\n' | evmdis
```
(`tr` for translate, `-d` for delete aka translate all '\n' chars into nothing)
